src/style.css

```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.image {
+  margin: 32px auto 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  display: flex;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+}
+
+.name {
+  margin: 0 auto 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  display: flex;
+}
+
+.code {
+  margin-left: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
in this project should be 2 BEM blocks in your code - card and stars. create new folder inside src, for example "styles". in styles folder create files "card.css" and "stars.css". paste all code , which related with stars in stars.css and import it in this file. same with card block